docs/standard/data/xml/xslt-transformations.md

```diff
@@ -18,7 +18,7 @@ ms.workload:
   - "dotnetcore"
 ---
 # XSLT Transformations
-The Extensible Stylesheet Language Transformation (XSLT) lets you transform the content of a source XML document into another document that is different in format or structure. For example, you can use XSLT to transform XML into HTML for use on a Web site or to transform it into a document that contains only the fields required by an application. This transformation process is specified by the [W3C XSL Transformations (XSLT) Version 1.0 recommendation](http://go.microsoft.com/fwlink/?LinkID=49919).  
+The Extensible Stylesheet Language Transformation (XSLT) lets you transform the content of a source XML document into another document that is different in format or structure. For example, you can use XSLT to transform XML into HTML for use on a Web site or to transform it into a document that contains only the fields required by an application. This transformation process is specified by the [W3C XSL Transformations (XSLT) Version 1.0 recommendation](https://www.w3.org/TR/xslt-10/).  
```

Comment:
Probably would be better to link to https://www.w3.org/TR/xslt/ and change the link text to "W3C XSL Transformations (XSLT) recommendation" without version to be future-proof